components/ide/jetbrains/cli/cmd/root.go

```diff
@@ -15,10 +17,21 @@ var rootCmd = &cobra.Command{
 }
 
 func Execute() {
+	waitUntilBackendPluginIsAccessible()
 	err := rootCmd.Execute()
 	if err != nil {
 		os.Exit(1)
 	}
 }
 
+func waitUntilBackendPluginIsAccessible() {
+	for {
+		resp, httpError := http.Get("http://localhost:63342/api/gitpod/cli?op=ping")
```

Comment:
It can race no? after successful ping a client can drop and then command will fail

why don't we do a request which does not timeout and resolve await on server only?

Reply:
adding port as a parameter makes sense for testing, you don't need prev env for that

Reply:
Added the support for it to work on Dev Server.
I've also added one line to `launch-dev-server.sh`.